tagging_tools/elf_loader.h

```diff
@@ -43,12 +43,21 @@ struct file_stream_t {
   virtual bool seek(size_t where, whence_t whence) = 0;
 };
 
+#ifdef RV64_VALIDATOR
```

Comment:
Is there a way we can simpify this to not use the `ifdef`?

Reply:
Not as easily.  Preferably the elf handing code should be able to handle both 32/64-bit files based on which elf header is passed in, but that's a bit more work.  (It does look like the elf handling code might have been borrowed from a program that did dynamically parse both formats, so finding the original inspiration for the code might help make this change easier.)